server/gokbg3/src/main/groovy/org/gokb/CrossRefPkgRun.groovy

```diff
@@ -178,9 +180,23 @@ class CrossRefPkgRun {
           }
         }
 
-        if (currentTippError.size() > 1) {
+
+        if (invalidTipps.contains(json_tipp)) {
+          reviewRequestService.raise(
+            pkg,
+            "TIPP rejected",
+            "TIPP ${json_tipp.name ?: json_tipp.title.name} coudn't be imported. ${(currentTippError as JSON).toString()}",
```

Comment:
The error object in itself should probably go into the `additionalInfo` JSON field for later output processing, rather than adding it to a user-facing message String.

Reply:
additionalInfo has a very complicated way of showing the Data to the User that needs further developments on the MVC view end. this is the simplified solution, until a structured way of handling it is defined. Same in the Job Overview.

Reply:
putting the errorMap in addtionalInfo didn't led to showing the Error in the (Legacy) UI

Reply:
For a structured display in the template UI, the corresponding [view](https://github.com/openlibraryenvironment/gokb/blob/g3_master/server/gokbg3/grails-app/views/apptemplates/_revreq.gsp) would need to be extended. Currently only a very specific JSON format under `additionalInfo.problems` is processed (starting on line 136), which is not compatible with the error format we are generating here.

Reply:
yeah, I know. it it exclusively used by the TSVIngestionService for one single type of errors from an unused identifier match, including suggestions on how to prevent the error. For the multitude of errors (and structures) that the import is able to generate, a proper definition of what and how to implement that view seems necessary.